packages/material-ui/src/FormControlLabel/FormControlLabel.js

```diff
@@ -74,6 +79,7 @@ function FormControlLabel(props, context) {
       className={classNames(
         classes.root,
         {
+          [classes.start]: labelPlacement === 'start',
```

Comment:
What're the pros/cons of using CSS over changing the DOM structure?

Reply:
Cons: confusing output when debugging in the console. But I'm fine with the current tradeoff.